packages/zero-client/src/client/custom.test.ts

```diff
@@ -1,8 +1,17 @@
-import {expect, expectTypeOf, test} from 'vitest';
+import {beforeEach, describe, expect, expectTypeOf, test} from 'vitest';
 import {schema} from '../../../zql/src/query/test/test-schemas.ts';
-import type {CustomMutatorDefs, MakeCustomMutatorInterfaces} from './custom.ts';
+import {
+  TransactionImpl,
+  type CustomMutatorDefs,
+  type MakeCustomMutatorInterfaces,
+  type Transaction,
+} from './custom.ts';
 import {zeroForTest} from './test-utils.ts';
 import {nanoid} from '../util/nanoid.ts';
+import {createDb} from './ivm-source-repo.test.ts';
```

Comment:
Do not import from *.test.ts files because it will cause those beforeEach, test, describe etc to run when this test file is run.

Can you update our eslint config to disallow this?

Instead move test util functions to test-util.ts.

Reply:
https://github.com/rocicorp/mono/pull/3720